symfony/webpack-encore/0.9/webpack.config.js

```diff
@@ -0,0 +1,23 @@
+var Encore = require('@symfony/webpack-encore');
+
+Encore
+    // directory where all compiled assets will be stored
+    .setOutputPath('web/build/')
+    // what's the public path to this directory (relative to your project's document root dir)
+    .setPublicPath('/build')
+    // empty the outputPath dir before each build
+    .cleanupOutputBeforeBuild()
+    // allow sass/scss files to be processed
+    .enableSassLoader()
```

Comment:
There are some issues with node-sass regarding segfaults on many systems (7 issues about segfaults on [sass/node-sass](https://github.com/sass/node-sass/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aopen%20segmentation)) , why not recommend `less` instead of `sass` by default to avoid people arguing Encore not working with `sass` (when it's obviously a `node-sass` issue)?